config/main.js

```diff
@@ -4,7 +4,7 @@ const { archivesDir } = require('../dist/filePaths');
 /** @type { import('@storybook/server-webpack5').StorybookConfig } */
 const config = {
   stories: [path.resolve(archivesDir(), '*.stories.json')],
-  addons: ['@storybook/addon-essentials', '../dist'],
```

Comment:
We do need this if we want to support viewport switching in the storybook. Well, maybe we only need that one addon, but does it hurt to have all of these?